src/styles/_mixins.scss

```diff
@@ -19,3 +19,31 @@
     }
   }
 }
+
+@mixin generateCardClasses() {
+  @each $cardType, $cardColor in $cardTypes {
+    .card--#{$cardType}.card {
```

Comment:
To nie jest poprawny modyfikator, wypluje Ci coś w stylu `.card--success.card`. W poprawnym BEM-ie to powinno być po prostu `.card--success` by nie zwiększać _specificity_.